js/coinfalcon.js

```diff
@@ -121,10 +121,6 @@ module.exports = class coinfalcon extends Exchange {
                     'taker': 0.002, // tiered fee starts at 0.2%
                 },
             },
-            'precision': {
-                'amount': this.parseNumber ('0.00000001'),
-                'price': this.parseNumber ('0.00000001'),
-            },
             'precisionMode': TICK_SIZE,
```

Comment:
same here, it was probably copied from other exchange, as exchange-wide precision is not needed here (because `fetchMarkets` already have precisions, which is superior)